3.8/arangosearch-geospatial-search.md

```diff
@@ -1,13 +1,324 @@
 ---
 layout: default
-description: ArangoSearch supports geospatial queries to find coordinates and GeoJSON shapes within a radius or area
+description: ArangoSearch supports geospatial queries like finding coordinates and GeoJSON shapes within a radius or area
 title: Geospatial Search ArangoSearch Examples
 ---
 # Geospatial Search with ArangoSearch
 
 {{ page.description }}
 {:class="lead"}
 
+ArangoSearch can accelerate various types of geospatial queries for data that
+is indexed by a View. The regular [geospatial index](indexing-geo.html) can do
+most of this too, but ArangoSearch allows you to combine geospatial requests
+with other kinds of searches, like full-text search.
 
+## Creating geospatial Analyzers
 
-**Dataset:** [IMDB movie dataset](arangosearch-example-datasets.html#imdb-movie-dataset)
+Geospatial data that can be indexed:
+- GeoJSON features such as Points and Polygons
+  (with coordinates in `[longitude, latitude]` order)
+- Raw coordinates (`[latitude, longitude]` arrays or two separate attributes)
+
+The data needs to be pre-processed with a `geojson` Analyzer in case of GeoJSON
+and with a `geopoint` Analyzer in case of raw coordinates. These Analyzers need
+to be created manually.
+
+**Custom Analyzers:**
+
+Create a `geojson` Analyzer in arangosh to pre-process arbitrary GeoJSON shapes.
+The default properties are usually what you want, therefore an empty object
+is passed:
+
+```js
+//db._useDatabase("your_database"); // Analyzer will be created in current database
+var analyzers = require("@arangodb/analyzers");
+analyzers.save("geojson", "geojson", {}, ["frequency", "norm", "position"]);
+```
+
+See [`geojson` Analyzer](analyzers.html#geojson) for details.
+
+Create a `geopoint` Analyzer in arangosh to pre-process raw coordinate arrays
+using the default properties, hence passing an empty object:
+
+```js
+//db._useDatabase("your_database"); // Analyzer will be created in current database
+var analyzers = require("@arangodb/analyzers");
+analyzers.save("geo_pair", "geopoint", {}, ["frequency", "norm", "position"]);
+```
+
+Create a `geopoint` Analyzer in arangosh to pre-process raw coordinates with
+latitude and longitude stored in two different attributes. These attributes
+cannot be at the top-level of the document, but must be nested in an object,
+e.g. `{ location: { lat: 40.78, lon: -73.97 } }`. The path relative to the
+parent attribute (here: `location`) needs to be described in the Analyzer
+properties for each of the coordinate attributes:
+
+```js
+//db._useDatabase("your_database"); // Analyzer will be created in current database
```

Comment:
leftover?

Reply:
No, this is on purpose to remind users that they may want to change to the appropriate database.